cardinal/system/system_game_setup.go

```diff
@@ -198,6 +227,112 @@ func (s *GameSetup) setupMountainPath(world cardinal.WorldContext) {
+		"window", plainBarn, world)}
+
+	// Petrol Object
+	burn := s.createAction(enums.ActionTypeBurn, " The petrol (with its unique funny smell)"+
+		" burns fiercely with a scent that makes you feel dizzy", true, false, true, 0, 0)
+
+	petrolActions := []uint32{burn}
+	objs := []component.Object{s.createObject(enums.ObjectTypePetrol, enums.MaterialTypeIKEA,
+		"a strange liquid that seems to be petrol, probably there is three liters of it, it's highly burnable though", enums.ObjectTypePetrol.String(), petrolActions, true, world)}
+
+	// Matchsticks Object
+	light := s.createAction(enums.ActionTypeLight, " The matchsticks (despite their small size)"+
+		" lights enough to see a small distance. You have to use them quickly"+
+		" or your fingers will be burn", true, false, true, 0, 0)
+
+	matchesActions := []uint32{light}
+	objs = append(objs, s.createObject(enums.ObjectTypeMatchsticks, enums.MaterialTypeWood,
+		"a group of matchsticks that have survived the pass of time, you can probably light them up, as they seem to be in a good condition", enums.ObjectTypeMatchsticks.String(), matchesActions, true, world))
+
+	roomID := s.RoomStore.Add(component.Room{
+		Description: "a dusty forge",
+		RoomType:    enums.RoomTypeForge,
+	})
+
+	tidForge := s._textGuid("a dusty forge")
+	txtForge := (" and you can see that it has not been used in ages." +
+		" There are many blood spots accross the forge and even the anvil is broken." +
+		" You don't know what happened here.")
+
+	// Logs to verify the ID's in the setup
+	world.Logger().Debug().Msgf("Object IDs for room %d: %v", roomID, [32]uint32{})
+	world.Logger().Debug().Msgf("Directional Object IDs for room %d: %v", roomID, dObjs)
+
+	// Inspect Data Passed to Setup Functions
+	world.Logger().Debug().Msgf("Data passed to createPlace for room %d: %+v", roomID, [32]uint32{})
+	world.Logger().Debug().Msgf("Data passed to createDirectionalObject for room %d: %+v", roomID, dObjs)
+
+	s.createPlace(roomID, enums.RoomTypeForge, dObjs, objs, tidForge, txtForge, world)
+
+	world.Logger().Debug().Msg("---->Forge setup complete")
+}
+
+// setupCellar sets up the cellar room in the game world.
+func (s *GameSetup) setupCellar(world cardinal.WorldContext) {
+	// KCELLAR -> UP
+	world.Logger().Debug().Msg("---->Setting up the cellar.....")
+
+	open2Barn := s.createAction(enums.ActionTypeOpen, " The way is opened"+
+		" just don't fall when going up", true, true, true, 0, 0)
+	plainBarn := []uint32{open2Barn}
+	dObjs := []component.Object{s.createDirObject(enums.DirectionTypeUp, enums.RoomTypeBarn,
+		enums.ObjectTypeTrapdoor, enums.MaterialTypeStone,
+		"trapdoor", plainBarn, world)}
+
+	// Dynamite Object
+	lightDynamite := s.createAction(enums.ActionTypeLight, " Seeing the fuse sparkling makes you remember the fireworks you loved as a child."+
+		" Suddenly, you return back and see that there is almost no time."+
+		" You have to do something or you will be flying into meat pieces!", true, false, true, 0, 0)
```

Comment:
" you have to do something or you will be turned to meat puree"

Reply:
done